core/src/main/java/com/yahoo/ycsb/BasicDB.java

```diff
@@ -111,21 +124,23 @@ public Status read(String table, String key, Set<String> fields, HashMap<String,
 
 		if (verbose)
 		{
-			System.out.print("READ "+table+" "+key+" [ ");
+      StringBuilder sb = getStringBuilder();
```

Comment:
Is this trying to save the string builder allocation that the existing code causes? Is the difference quantifiable?